cmd/broker/broker.go

```diff
@@ -49,6 +54,7 @@ func init() {
 }
 
 func main() {
+	log.Printf("Broker starting with version %s, commit %s", version, commit)
```

Comment:
This will yield an info level log message with version and commit as k/v pairs, easily digestible by any log aggregator. This would be more consistent with most other log messages from OSBA:

```
log.WithFields(
  log.Fields{
    "version": version,
    "commit":  commit,
  },
).Info("Open Service Broker for Azure starting")
```